.github/workflows/bridget_update.yml

```diff
@@ -0,0 +1,35 @@
+name: Create PR in android-news-app when bridget is updated
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build and copy bridget.jar
+    runs-on: 4core-ubuntu-latest
+    timeout-minutes: 30
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up JDK 17
+      uses: actions/setup-java@v3
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+
+    - name: Setup Gradle
+      uses: gradle/gradle-build-action@v2
+
+    - name: Build jar
+      run: |
+        ./gradlew :library:build
+    
+    - name: Add jar to ANA
+      env:
+        GUARDIANANDROID_PAT: ${{ secrets.GUARDIANANDROID_PAT }}
```

Comment:
This secret doesn't exist yet. @jorgeazevedo Can you add the secret to this repo?

Reply:
@mohammad-haque I need the secret added in this repo.